app/src/main/java/com/sun/imageeditor/utils/base/BaseActivity.kt

```diff
@@ -22,4 +22,24 @@ abstract class BaseActivity<VB : ViewBinding>(
 
     abstract fun initView()
     abstract fun initData()
+
+    override fun onBackPressed() {
+        try {
+            var handled = false
+            supportFragmentManager.fragments.forEach { fragment ->
+                if (fragment is BaseFragment<*>) {
+                    handled = fragment.onBackPressed()
+                    if (handled) {
+                        return
+                    }
+                }
+            }
+
+            if (!handled) {
+                super.onBackPressed()
+            }
```

Comment:
fragment nào đang hiển thị kích hoạt onBackPress ở fragment đó mới đúng chứ. Như này sẽ kích hoạt toàn bộ onBackPress của các fragment

Reply:
ở đây có
`if (handled) {
  return
}`
rồi nên nó sẽ dừng khi fragment nào trả onBackPress true mà nhỉ